### PR TITLE
added GetBalance

### LINKED ISSUE #696: Create method to get balance of wallet in list of wallets
We need to be able to see the balance of the wallets in the list that user sees before selecting wallet to recover from.

### PR DESCRIPTION
**Pull request by dabasov**
### Changes
-

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: zcncore/wallet_mobile.go [hunk 1]
```diff
@@ -22,6 +26,14 @@ func (w *wallet) Sign(hash string) (string, error) {
 	return sigScheme.Sign(hash)
 }
 
+func GetWalletBalanceMobile(id string) (string, error) {
```
Comment (cnlangzi) [inline zcncore/wallet_mobile.go]:
```suggestion
func GetWalletBalance(id string) (int64, error) {
```

**Review by cnlangzi — COMMENTED** [review]
`GetWalletBalance` should be ok

File: zcncore/wallet_mobile.go [hunk 2]
```diff
@@ -22,6 +26,14 @@ func (w *wallet) Sign(hash string) (string, error) {
...
+	balance, err := GetWalletBalance(id)
+	if err != nil {
+		return "", err
+	}
+	return strconv.FormatInt(int64(balance), 10), nil
```
Comment (cnlangzi) [inline zcncore/wallet_mobile.go]:
```suggestion
return int64(balance), nil
```

**Review by cnlangzi — COMMENTED** [review]
`int64` works on gombile

**Review by peterlimg — CHANGES_REQUESTED** [review]
@dabasov please update as suggested by @cnlangzi

File: zcncore/wallet_mobile.go (hunk 2 above)
Comment (dabasov) [reply zcncore/wallet_mobile.go]: last time you told that it is not working

Comment (cnlangzi) [reply zcncore/wallet_mobile.go]: uint64 does't work on gomobile, but int64 works.

**Comment by cnlangzi** [comment]
@peterlimg  I have got them fixed. please review it again. thanks. cc @dabasov

File: zcncore/wallet_mobile.go (hunk 2 above)
Comment (cnlangzi) [reply zcncore/wallet_mobile.go]: fixed

File: zcncore/wallet_mobile.go (hunk 1 above)
Comment (cnlangzi) [reply zcncore/wallet_mobile.go]: fixed

**Review by cnlangzi — APPROVED** [review]

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**